activities/QRCode.activity/js/activity.js

```diff
@@ -210,9 +210,11 @@ define(["sugar-web/activity/activity","sugar-web/datastore", "sugar-web/env", "w
 
 		// Add entry to history
 		function addToHistory(text) {
+			if(!history.includes(text)){
 			history.push(text);
 			updateHistory();
 		}
+		}
```

Comment:
Could you realign code to respect indentation?

Reply:
done @llaske

Reply:
No you don't. Of course, line 213 and 214 must be aligned with block.

Reply:
Is it fine now? @llaske